agent/consul/client_serf.go

```diff
@@ -34,13 +34,17 @@ func (c *Client) setupSerf(conf *serf.Config, ch chan serf.Event, path string) (
 		conf.Tags["acls"] = string(structs.ACLModeDisabled)
 	}
 
+	const subLoggerName = "lan"
```

Comment:
What is the reasoning behind storing these here vs in the `logging/names.go`

Reply:
Fixed